### PR TITLE
[OPERATOR-502] Add support to check portworx-operator version

### PR DESCRIPTION
**Pull request by pure-jliao**
**What this PR does / why we need it**: Check the px-operator version and skip tests

**Which issue(s) this PR fixes** (optional)
Closes #

**Special notes for your reviewer**:

### CODE REVIEW

File: test/integration_test/basic_test.go [hunk 1]
```diff
@@ -100,6 +100,9 @@ var testStorageClusterBasicCases = []types.TestCase{
 			return cluster
 		},
 		TestFunc: InstallWithTelemetry,
+		ShouldSkip: func(tc *types.TestCase) bool {
+			return ci_utils.PxOperatorVersion.LessThanOrEqual(ci_utils.PxOperatorVer1_6)
```
Comment (haibinxie) [inline test/integration_test/basic_test.go:104]: 1.6.1 will also not have it.

**Review by haibinxie — APPROVED** [review]
good to merge after resolving the one comment.

File: test/integration_test/basic_test.go (hunk 1 above)
Comment (pure-jliao) [reply test/integration_test/basic_test.go]: I planned to use 1.7 for checking that, but we may have different tags like 1.7.0-dev, 1.7.0-rc1. did some experiments seems `1.7-` should do the job,

File: test/integration_test/main_test.go [hunk 2]
```diff
@@ -63,6 +64,11 @@ func setup() error {
 
 	ci_utils.PxUpgradeHopsURLList = strings.Split(pxUpgradeHopsURLs, ",")
 
+	ci_utils.PxOperatorVersion, err = ci_utils.GetPXOperatorVersion()
+	if err != nil {
+		return fmt.Errorf("failed to discover installed portworx operator version: %v", err)
+	}
+
```
Comment (piyush-nimbalkar) [inline test/integration_test/main_test.go:71]: nit: If we are setting the variable `PxOperatorVersion` using a function also in the same package, should we just call `ci_utils.SetPxOperatorVersion` or `ci_utils.InitPxOperatorVersion` and let it set the value inside the function itself?

**Review by piyush-nimbalkar — APPROVED** [review]
LGTM
Minor comment. But feel free to merge without it.

File: test/integration_test/main_test.go (hunk 2 above)
Comment (pure-jliao) [reply test/integration_test/main_test.go:71]: the operator right now is installed out of the automation, so Get may work better for now. in the future if we install operator in the test, probably will change it to set.

**Pull request merged**